cloudconfig/podcfg/image.go

```diff
@@ -103,6 +92,16 @@ func tagImagePath(fullPath, tag string) (string, error) {
 	return imageNamed.String(), nil
 }
 
+// ImageRepoToPath returns the OCI image path.
```

Comment:
I'll need to come back and blow away all this "path" language as it is confusing.